test/over_react_test/helper_components/sample_component.dart

```diff
@@ -17,10 +17,16 @@ class _$SampleProps extends UiProps {
 
   bool shouldAlwaysBeFalse;
 
-  bool shouldError;
+  bool shouldErrorInRender;
+
+  bool shouldErrorInMount;
+
+  bool shouldErrorInUnmount;
 
   bool addExtraLogAndWarn;
 
+  Function() componentDidMountCallback;
```

Comment:
#supernit (you definitely don't need to change this)... but I thought I would point out that - for callbacks, our naming convention is usually `on{ThingThatWillOrDidHappen}`. Its a small change, but IMO - it makes it more clear / "plain english" about what the function does.  Its also shorter than spelling out Callback :)

```suggestion
  Function() onComponentDidMount;
```

Reply:
That is a lot better, and I always appreciate being reminded of our conventions!